src/content/release/breaking-changes/RenderConstrainedLayoutBuilder-mixin.md

```diff
@@ -0,0 +1,109 @@
+---
+title: RenderConstrainedLayoutBuilder renamed RenderAbstractLayoutBuilderMixin
+description: >-
+  'RenderConstrainedLayoutBuilder' is replaced by
+  'RenderAbstractLayoutBuilderMixin' with a
+  new 'layoutInfo' getter to implement.
+---
+
+## Summary
```

Comment:
From what I remember this sounds all correct. I do wonder, though, if we should re-add a class `RenderConstrainedLayoutBuilder` with a default implementation of `layoutInfo` to reduce churn and make this not breaking?

Reply:
Oh way better, let me look into this today. Thank you @goderbauer!

Reply:
I think we managed to un-break this. 🎊 
https://github.com/flutter/flutter/pull/167689